api/tests/core/users/test_api.py

```diff
@@ -2115,6 +2167,8 @@ def test_anonymize_beneficiary_users(self) -> None:
             assert user_to_anonymize.action_history[0].actionType == history_models.ActionType.USER_ANONYMIZED
             assert user_to_anonymize.action_history[0].authorUserId == None
 
+            assert len(os.listdir(self.storage_folder)) == 0
```

Comment:
```suggestion
        assert len(os.listdir(self.storage_folder)) == 0
```
Pas besoin de le mettre dans la boucle vu que ça ne dépend pas du user a anonymiser.

D'ailleurs il vaudrait mieux le faire avant la boucle.